Source/p_setup.c

```diff
@@ -324,17 +324,17 @@ void P_LoadNodes (int lump)
           no->children[j] = (unsigned short)SHORT(mn->children[j]); // [FG] extended nodes
 
           // [FG] extended nodes
-          if (no->children[j] == 0xFFFF)
+          if (no->children[j] == (unsigned short)-1)
```

Comment:
Maybe use `NO_INDEX` as defined in the header? Or is that for a different meaning?

Reply:
`NO_INDEX` is meant for sidedefs, although it's the same value under the hood.